docs/spark/tutorials/get-started-macos-instructions.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Get started with .NET for Apache Spark on macOS
+description: Discover how to run a .NET for Apache Spark app using .NET Core on MacOSX.
+ms.date: 01/22/2019
+ms.topic: tutorial
+ms.custom: mvc
+# Customer intent: As a developer, I want to write a simple custom application using .NET for Apache Spark on MacOSX.
+---
+
+# Tutorial: Get started with .NET for Apache Spark on macOS
+
+This tutorial shows you how to run a .NET for Apache Spark app using .NET Core on macOS.
+
+> [!div class="checklist"]
+>
+> * Prerequisites to run a .NET for Apache Spark application
+> * Write a simple .NET for Apache Spark application
+> * Build and run a .NET for Apache Spark application
+
+## Prerequisites
+
+- Download and install **[.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)** 
```

Comment:
If we do decide to have this as a separate article from the current Get Started, take a look at how we worded things there *and* in Niharika's PR for this pre-req section to make sure it's consistent.